autogen/telemetry.py

```diff
@@ -116,22 +116,18 @@ def get_connection():
 def _to_dict(
     obj: Union[int, float, str, bool, Dict[Any, Any], List[Any], Tuple[Any, ...], Any], exclude: List[str] = []
 ) -> Any:
+    from autogen import Agent
```

Comment:
to avoid circular dependency

Reply:
Curious why would there be circular dependency? the `autogen.agentchat.agent.Agent` doesn't import this module.

Reply:
ConversableAgent and GroupChat need to import Telemetry and ConversableAgent inherits Agent which create the circular dependency.